samples/sensor/bq274xx/prj.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_I2C=y
+CONFIG_SENSOR=y
+CONFIG_BQ274XX=y
+
+CONFIG_LOG=y
+CONFIG_LOG_IMMEDIATE=y
```

Comment:
is this really needed?

Reply:
Not needed as it is not consumed. Removed, thanks